protobuf/lib/src/protobuf/coded_buffer_writer.dart

```diff
@@ -386,15 +391,16 @@ class CodedBufferWriter {
         break;
       case PbFieldType._MESSAGE_BIT:
         final mark = _startLengthDelimited();
+        value as GeneratedMessage;
         value.writeToCodedBufferWriter(this);
         _endLengthDelimited(mark);
         break;
     }
   }
 
-  void _writeBytesNoTag(dynamic value) {
-    writeInt32NoTag(value.length);
-    writeRawBytes(value);
+  void _writeBytesNoTag(Object value) {
```

Comment:
I suggest changing this to `TypedData value` and fixing callers. One caller is already properly typed the second one (`PbFieldType._STRING_BIT`) can be rewritten as `const Utf8Encoder().convert(str)` to make sure that we have `Uint8List`.

Reply:
Thanks! Done.